src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java

```diff
@@ -27,6 +27,41 @@ public static Network create(UUID uuid, boolean createHvdcLine) {
         return create(uuid, createHvdcLine, new NetworkFactoryImpl());
     }
 
+    public static Network createSwitchNetwork(UUID uuid, NetworkFactory networkFactory) {
```

Comment:
Add a comment to explain what's the point of this network: fictitious switch and different switch kind